src/Helpers/spotify.js

```diff
@@ -0,0 +1,38 @@
+export const authEndpoint = "https://accounts.spotify.com/authorize";
+const clientId = '3ed3b11c86b4444399c6be191dc7dd2f';
+const redirectUri = "http://localhost:3000/";
```

Comment:
Perhaps when you deploy this it can cause problems since when you want to do the redirect it will not find that domain, here is a small example of how to solve it

// config.js
`export const DEVENV = process.env.REACT_APP_DEVENV;`

// your component
```
import { DEVENV } from '../../config';
const redirectUri = DEVENV ? 'http://localhost:3000/': 'your_netlify_domain';
```

Run the app:
`npm run start REACT_APP_DEVENV = true`